service-api/module/Application/src/Model/Service/Pdfs/Service.php

```diff
@@ -168,7 +168,10 @@ private function getPdfStatus(Lpa $lpa, $type)
             // If we get here it exists in the bucket...
             return self::STATUS_READY;
         } catch (\Aws\S3\Exception\S3Exception $ignore) {
-            $this->getLogger()->error('Exception while attempting to get PDF info from S3');
+            $this->getLogger()->error('Exception while attempting to get PDF info from S3', [
+                'error_code' => 'PDF_S3_HEAD_FAILED',
+                'exception' => $ignore,
+            ]);
```

Comment:
This is one of our most common log messages, because it happens even if there's simply a 404 (i.e. for any new PDFs), so I think it would be helpful to exclude that:

```suggestion
            if ($e->getStatusCode() !== 404) {
                $this->getLogger()->error('Exception while attempting to get PDF info from S3', [
                    'error_code' => 'PDF_S3_HEAD_FAILED',
                    'exception' => $ignore,
                ]);
            }
```